### PR TITLE
logger.profile default level = verbose

### LINKED ISSUE #1436: Change default level for profiling logs
In winston logger, `logger.profile`, if level is not explicitly set, set to verbose.

**Edit:**
Log profiling logs to verbose, not info.

### PR DESCRIPTION
**Pull request by dapplion**
Fixes https://github.com/ChainSafe/lodestar/issues/1436

@tuyennhv Would this mess up your performance tests?

### CODE REVIEW

File: packages/lodestar-utils/src/logger/winston.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export class WinstonLogger implements ILogger {
   }
 
   profile(message: string, option?: {level: string; message: string}): void {
-    this.winston.profile(message, option);
+    this.winston.profile(message, {level: LogLevel.verbose, ...option} as LogEntry);
```
Comment (wemeetagain) [inline packages/lodestar-utils/src/logger/winston.ts:83]: I seem to remember that this winston didn't respect the log level here. Haven't tested this out, so not sure if thats still the case.

**Comment by twoeths** [comment]
> Fixes #1436
> 
> @tuyennhv Would this mess up your performance tests?

we can either set default log level to `verbose` for the performance tests (`test:perf` in package.json) or update every `logger.profile` to use info level.

**Comment by wemeetagain** [comment]
bump

**Comment by dapplion** [comment]
Not necessary since we don't use profile in anywhere else than tests

**Pull request closed without merging**